MovieDB/Model/TrendingResults.swift

```diff
@@ -11,12 +11,12 @@ import SwiftUI
 
 struct TrendingResults: Decodable {
```

Comment:
This looks similar with Movie Model. 
Do you think it is possible to use the same Model?